src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,19 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    Machine trackOne = new Track();
```

Comment:
you should create main method here and do all of this there

Reply:
Fixed, now should be correct